query-engine/query-engine-node-api/src/logger.rs

```diff
@@ -24,10 +16,6 @@ use tracing_subscriber::{
 };
 
 pub(crate) struct Logger {
-    // We need the original parameters in case we move to tracing
-    log_queries: bool,
-    log_level: LevelFilter,
-    log_callback: ThreadsafeFunction<String>,
     __dispatcher: RwLock<Dispatch>,
```

Comment:
Do we still need the rwlock? we not changing the dispatcher based on if tracing is happening or not. We probably can go back to creation of dispatch on instantiaion.

Reply:
Good finding, the `RwLock` can disapear. I will keep the Logger in its separate struct, in that way would be easier to expand it if needed in the future.